src/dsymbol.d

```diff
@@ -183,7 +183,7 @@ enum : int
                                     // meaning don't search imports in that scope,
                                     // because qualified module searches search
                                     // their imports
-    SearchCheck10378        = 0x40, // unqualified search with transition=checkimports switch
+    //SearchCheck10378        = 0x40, // unqualified search with transition=checkimports switch
```

Comment:
Since we use a version control system, what's the motivation to comment it instead of removing it ?